### ISSUE TITLE
Content-Type error

### ISSUE THREAD

**Issue by bondjw07** [AustinHasten/PlexHolidays#9]
Getting this error when almost done processing the library.  Any suggestions?  I'm guessing it's a bad file, but I'm not sure how to know what file it is.
`TV Shows:  98%|████████████████████████████▍| 7109/7253 [26:31<01:00,  2.38it/s]Traceback (most recent call last):
  File "__init__.py", line 117, in <module>
    ph = PlexHolidays()
  File "__init__.py", line 62, in __init__
    self.results = ThreadPool(10).map(self.find_matches, self.plex.media)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 290, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 683, in get
    raise self._value
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "__init__.py", line 78, in find_matches
    imdb_id = self.get_imdb_id(plex_guid)
  File "__init__.py", line 92, in get_imdb_id
    return self.get_episode_id(plex_guid)
  File "__init__.py", line 99, in get_episode_id
    episode = self.get_tvdb_series(series_id)[season][episode]
  File "<decorator-gen-6>", line 2, in get_tvdb_series
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "__init__.py", line 107, in get_tvdb_series
    return self.tvdb.get_series(series_id, 'en')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/utils.py", line 47, in __wrapper__
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/api.py", line 729, in get_series
    data = self.loader.load(url, cache)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/loader.py", line 73, in load
    if response['content-type'] == "application/zip":
KeyError: 'content-type'
`

**Comment by bondjw07** [comment]
I tried it again (after deleting some unnecessary Shows) and got the same error I had the very first time I ran it.  This time pointing to some garbage data and an error that reads "BadStatusLine".

`TV Shows:  96%|███████████████████████████▉ | 6529/6778 [25:41<00:16, 14.70it/s]Traceback (most recent call last):
  File "__init__.py", line 117, in <module>
    ph = PlexHolidays()
  File "__init__.py", line 62, in __init__
    self.results = ThreadPool(10).map(self.find_matches, self.plex.media)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 290, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 683, in get
    raise self._value
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "__init__.py", line 78, in find_matches
    imdb_id = self.get_imdb_id(plex_guid)
  File "__init__.py", line 92, in get_imdb_id
    return self.get_episode_id(plex_guid)
  File "__init__.py", line 99, in get_episode_id
    episode = self.get_tvdb_series(series_id)[season][episode]
  File "<decorator-gen-6>", line 2, in get_tvdb_series
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "__init__.py", line 107, in get_tvdb_series
    return self.tvdb.get_series(series_id, 'en')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/utils.py", line 47, in __wrapper__
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/api.py", line 729, in get_series
    data = self.loader.load(url, cache)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pytvdbapi/loader.py", line 64, in load
    response, content = self.http.request(url, headers=header)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/httplib2/__init__.py", line 1882, in request
    cachekey,
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/httplib2/__init__.py", line 1595, in _request
    conn, request_uri, method, body, headers
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/httplib2/__init__.py", line 1533, in _conn_request
    response = conn.getresponse()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 278, in _read_status
    raise BadStatusLine(line)
http.client.BadStatusLine: äº!wY(ìÎ`iñî
èa´¶¤òZH¼{õÄ´,_ÃÑó(%£fÂÄ>Òâ`iÔ@8³Z.»£<E[\Âê	¿Añ
                                                2
                                                  ÆKÑøU´ß»:^¸	íäökÕ÷|R,rdÕÐ~'Í~ +V^5Ñª(ê1øÖ,Àºq&ðk$÷õDC¦H¬*òüÜèôzµQS½§Ä^så÷^À{	g
                                                                         sãÖH\+¸:Ap5»d×v,Ä$>P~ýFÑÀë(Ib£KÄ,5=°yÊðVñ2×¨p~,ÍXíÊÂÚ	ªJ.,Ëñiòfù+PU¸

TV Shows:  96%|███████████████████████████▉ | 6530/6778 [25:42<00:58,  4.23it/s]`

**Comment by bondjw07** [comment]
Ok, this is pretty weird.  To debug the issue, I added the following line of code on line 79 to print out the `imdb_id` value after it was retrieved:
`#print(imdb_id)`
After doing that, the script ran fine.  I was thinking that maybe this resolved a race condition since it would have made the code run a bit slower.  So I removed it and tried it again, and it worked as expected.  So I'm going to go ahead and close this.  I have no idea why it started working, but I'm very happy with the results.  Thanks so much for this!
